static/css/newcommunity.css

```diff
@@ -244,7 +244,7 @@ a {
   display: block;
   margin: auto;
   margin-top: 2rem;
-  width: 156px;
+  width: 164px;
```

Comment:
@Arhell 👋  At 164px, I still see a two-line button, both locally and in the Netlify preview:

![Screen Shot 2020-11-16 at 11 59 16 AM](https://user-images.githubusercontent.com/3210446/99303140-47d4ad80-2805-11eb-8302-346a20c6b396.png)

It looks like the minimum width for button text on a single line is 166px.

![Screen Shot 2020-11-16 at 12 10 42 PM](https://user-images.githubusercontent.com/3210446/99303894-7b640780-2806-11eb-9370-80184f017dec.png)

Reply:
Hi @Arhell . Viewing from a smallish laptop, I see the two lines of text when my browser reaches the maximum width of the screen. At smaller widths, the button text displays in a single line.
Is there a better way of setting the width of the button to adjust for the size of the container (screen size)?

Reply:
done
Made it independent of the width
@zacharysarah 
@kbhawkey 
Thanks!